webrecorder/browsertrix
# Frontend prod build optimizations

- Leverage webpack chunk splitting to creating more, smaller JS files rather than one large main file (`import(file)` syntax)
- Enable long-term caching by adding content hash to output file names
- Copy entire `/dist` folder contents in Dockerfile
- Changed `yarn start-dev` -> `yarn start` since there is no prod server

### Manual testing
No functional changes.

First, install dev dependencies with `yarn`
1. Run app with `yarn start`
2. Navigate around app. It should work as expected
3. Stop server and run `yarn build`
4. Serve `dist` using method of choice. I use `python -m SimpleHTTPServer 8000`. App should work as expected

### TODO
- Handle FOUC
- https://github.com/ikreymer/browsertrix-cloud/issues/55
- Separate entry with index.html for each page? (check nginx SPA settings)
